src/coreclr/jit/llvm.cpp

```diff
@@ -2279,7 +2328,8 @@ void Llvm::ConvertShadowStackLocalNode(GenTreeLclVarCommon* node)
     GenTreeLclVarCommon* lclVar = node->AsLclVarCommon();
     LclVarDsc* varDsc = _compiler->lvaGetDesc(lclVar->GetLclNum());
     genTreeOps oper = node->OperGet();
-    if (!canStoreLocalOnLlvmStack(varDsc))
+
+    if (!canStoreLocalOnLlvmStack(varDsc) && !independentCanBeReplacedWithItsField(varDsc))
```

Comment:
> !independentCanBeReplacedWithItsField(varDsc)

Why is the condition needed? At this point we should not encounter any `CanBeReplacedWithItsField` locals.

Reply:
You are right, I think I added that before moving the call to `lowerStoreLcl`.  

It originally happened for the `ResolveRuntimeTypeHandle`.  The parameter is a `RuntimeTypeHandle` a struct with a singled field `EETypePtr`, itself a struct with a single field `MethodTable*`.  `RuntimeTypeHandle` is false for `canStoreLocalOnLlvmStack`